README.md

```diff
@@ -48,3 +48,50 @@ We welcome feedback and suggestions from any Biodata Catalyst team on useful tes
 strengthen the stability of interaction between components.  Please email either lblauvel at ucsc.edu or 
 bhannafi at ucsc.edu.
 
+# Current State of Testing
+
+| From / To     | Terra                    | Gen3                     | Dockstore                | Seven Bridges            | PIC-SURE                 | HeLx |
+| ------------- |:------------------------:|:------------------------:|:------------------------:|:------------------------:|:------------------------:| ----:|
+| Terra         | :heavy_multiplication_x: | :heavy_check_mark:       | :heavy_check_mark:       | :heavy_multiplication_x: | Future                   |      |
+| Gen3          | TBD                      | :heavy_multiplication_x: | TBD?                     | TBD?                     |                          |      |
+| Dockstore     |                          |                          | :heavy_multiplication_x: |                          |                          |      |
+| Seven Bridges | :heavy_multiplication_x: | TBD?                     | TBD?                     | :heavy_multiplication_x: | Future                   |      |
+| PIC-SURE      |                          | TBD?                     |                          |                          | :heavy_multiplication_x: |      |
+| HeLx          | TBD?                     | TBD?                     |                          | TBD?                     |                          | :heavy_multiplication_x: |
+
+ - :heavy_check_mark: : Implemented (probably not comprehensively; additional testing suggestions are very welcome)
+ - :heavy_multiplication_x: : Not Applicable
```

Comment:
Consider three indicators:
  - (x) no integrations tested
  - (foo) some integrations tested
  - (check) all integrations tested

Reply:
After speaking earlier, will implement :white_check_mark: for partial, though "all integrations tested" is unlikely to ever happen I think (similar to 100% coverage being a rare beast indeed).